dev-requirements.txt

```diff
@@ -107,3 +107,4 @@ xmltodict==0.12.0
 yamllint==1.20.0
 zipp==0.5.1
 git+https://github.com/ONSdigital/es-functions.git
+#To trigger build again.
```

Comment:
a bit of dev left behind?

Reply:
Yeah just realised I'd left that in to trigger the build. Will remove it now.